src/server_manager/web_app/ui_components/outline-server-view.js

```diff
@@ -994,16 +1049,27 @@ export class ServerView extends DirMixin(PolymerElement) {
     this.dispatchEvent(makePublicEvent('ForgetServerRequested'));
   }
 
-  _computePaperProgressClass(isAccessKeyDataLimitEnabled) {
-    return isAccessKeyDataLimitEnabled ? 'data-limits' : '';
+  _activeDataLimitForKey(accessKey) {
+    if (!accessKey) {
+      // We're in app startup
+      return null;
+    }
+    return accessKey.dataLimit || (this.isDefaultDataLimitEnabled ? this.defaultDataLimit : null);
+  }
+
+  _computePaperProgressClass(accessKey) {
+    return this._activeDataLimitForKey(accessKey) ? 'data-limits' : '';
   }
 
   _getDataLimitsUsageString(accessKey) {
```

Comment:
This is hiding the dependency on localize, so it won't update on language change.

Reply:
Done in 67492a22a967c621796b3878b61831699324f2a4

Reply:
Done in 67492a22a967c621796b3878b61831699324f2a4